docker/airflow/Dockerfile

```
@@ -33,21 +34,19 @@ ENV AIRFLOW__LOGGING__REMOTE_LOG_CONN_ID=aws_default
 ENV AIRFLOW__LOGGING__REMOTE_BASE_LOG_FOLDER=s3://openverse-airflow-logs
 
 
-RUN apt-get update && apt-get -yqq upgrade && apt-get -yqq install \
+USER root
+RUN apt-get update && apt-get -yqq install \
     build-essential \
     libpq-dev \
     libffi-dev \
     && apt-get autoremove -y \
     && rm -rf /var/lib/apt/lists/*
-RUN useradd -m -d ${AIRFLOW_HOME} airflow && \
-    mkdir -p ${OUTPUT_DIR} && \
-    chown airflow:airflow ${OUTPUT_DIR}
+RUN mkdir -p ${OUTPUT_DIR} /home/airflow/.ipython && chown airflow ${OUTPUT_DIR} /home/airflow/.ipython
```

New lines helps the understanding of each statement:
```suggestion

RUN mkdir -p ${OUTPUT_DIR} /home/airflow/.ipython && \
    chown airflow ${OUTPUT_DIR} /home/airflow/.ipython
```
and maybe add a comment that is for retaining `ipython` history?

---

Done in 31988874c2ae6f7d9062d80a47c1d4ee341f5b0f!